Source/OCGoogleDirectionsAPI/Client/RequestURLCreator/OCDirectionsRequestURLCreatorJSON.m

```diff
@@ -302,6 +307,15 @@ - (void)appendTransitRoutingPreference:(OCDirectionsRequest *)request toString:(
     [string appendString:transitRoutingPreferenceString];
 }
 
+- (void)appendHeading:(OCDirectionsRequest *)request toString:(NSMutableString *)string {
+    [string appendString:kOCGoogleDirectionsRequestAttributeHeading];
```

Comment:
Should we skip acting the query parameter name if the `heading` is `nil`?

Perhaps something like this could work:
```objc
if (request.heading == nil) {
     return;
}
[string appendString:kOCGoogleDirectionsRequestAttributeHeading];
[string appendString:[self stringFromCLLocationDirection:request.heading]];
```